assets/js/googlesitekit/datastore/user/feature-tours.test.js

```diff
@@ -286,4 +350,51 @@ describe( 'core/user feature-tours', () => {
 			} );
 		} );
 	} );
+
+	describe( 'getLastDismissedAt', () => {
+		it( 'returns initial state (undefined) if there is no lastDismissedAt timestamp', () => {
+			const lastDismissedAt = registry.select( STORE_NAME ).getLastDismissedAt();
+			expect( lastDismissedAt ).toEqual( undefined );
+		} );
+
+		it( 'returns the lastDismissedAt timestamp if there is one', () => {
+			const timestamp = Date.now();
+			registry.dispatch( STORE_NAME ).receiveLastDismissedAt( timestamp );
+			expect( registry.select( STORE_NAME ).getLastDismissedAt() ).toEqual( timestamp );
+		} );
+
+		it( 'sets lastDismissedAt in the store if there is a value in the cache', async () => {
+			const timestamp = Date.now();
+			const { setItem, deleteItem } = CacheModule;
+
+			setItem( FEATURE_TOUR_CACHE_KEY, timestamp, { ttl: FEATURE_TOUR_COOLDOWN_PERIOD } );
+
+			registry.select( STORE_NAME ).getLastDismissedAt();
+			await untilResolved( registry, STORE_NAME ).getLastDismissedAt();
+
+			expect( store.getState().lastDismissedAt ).toEqual( timestamp );
+			await	deleteItem( FEATURE_TOUR_CACHE_KEY );
+		} );
+	} );
+
+	describe( 'areFeatureToursOnCooldown', () => {
+		it( 'returns undefined if there is no lastDismissedAt timestamp', () => {
+			expect( registry.select( STORE_NAME ).areFeatureToursOnCooldown() ).toBeUndefined();
+		} );
+
+		it( 'returns true if the lastDismissedAt timestamp is within the feature tour cooldown period', () => {
+			const timestamp = Date.now();
+			registry.dispatch( STORE_NAME ).receiveLastDismissedAt( timestamp );
```

Comment:
Let's use the `FEATURE_TOUR_COOLDOWN_PERIOD` here as well but tweak the resulting timestamp (+1 sec or so) so that it falls just inside the cooldown range rather than right after it's been dismissed. This way we're more likely to catch a bug if the TTL were to be updated somewhere without using the constant for example.